tools/packaging/static-build/agent/Dockerfile

```diff
@@ -26,16 +26,17 @@ RUN mkdir ${RUSTUP_HOME} ${CARGO_HOME} && chmod -R a+rwX /opt
 RUN apt-get update && \
 	apt-get --no-install-recommends -y install \
 		ca-certificates \
+		clang \
```

Comment:
IIUC this fixes build of the agent caused by the previous commit. Please squash the fix there for the sake of bisect.